crates/pop-parachains/src/errors.rs

```diff
@@ -25,12 +25,18 @@ pub enum Error {
 	ManifestError(#[from] pop_common::manifest::Error),
 	#[error("Missing binary: {0}")]
 	MissingBinary(String),
+	#[error("Missing chain spec file at: {0}")]
+	MissingChainSpec(String),
+	#[error("Command {command} doesn't exist in binary {binary}")]
+	MissingCommand { command: String, binary: String },
 	#[error("Orchestrator error: {0}")]
 	OrchestratorError(#[from] OrchestratorError),
 	#[error("Failed to create pallet directory")]
 	PalletDirCreation,
 	#[error("ParseError error: {0}")]
 	ParseError(#[from] url::ParseError),
+	#[error("{0}")]
+	PopCommonError(#[from] pop_common::Error),
```

Comment:
```suggestion
	CommonError(#[from] pop_common::Error),
```

We dont need to prefix it with Pop, the context of the crate is pop.